app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -233,6 +235,31 @@ void ControllerContainer::SetHandJointLocations(const int32_t aControllerIndex,
 
             controller.handMeshToggle->AddNode(transform);
         }
+
+#if !defined(OCULUSVR)
+        // Initialize left hand action button, which for now triggers back navigation
+        if (controller.leftHanded) {
+            TextureGLPtr texture = create->LoadTexture("menu.png");
+            assert(texture);
+            texture->SetTextureParameter(GL_TEXTURE_MAG_FILTER, GL_LINEAR);
+            texture->SetTextureParameter(GL_TEXTURE_MIN_FILTER, GL_LINEAR);
+            // texture->Bind();
+
+            const float width = 1.75f;
+            const float aspect = (float)texture->GetWidth() / (float)texture->GetHeight();
+
+            QuadPtr icon = Quad::Create(create, width, width / aspect, nullptr);
```

Comment:
This is a bit weird. Why do you compute the aspect? In the end you're passing width and height...